CHANGELOG.md

```diff
@@ -19,15 +19,14 @@
 - [#16169](https://github.com/emberjs/ember.js/pull/16169) [BUGFIX] Fix various issues with descriptor trap.
 - [#16174](https://github.com/emberjs/ember.js/pull/16174) [BUGFIX] Enable _some_ recovery of errors thrown during render.
```

Comment:
May be duplicated: found 3 other times:
- 2.18.1 (line 66)
- 2.17.1, (line 85)
- 2.16.3, (line 115)

Reply:
Hmm, that isn't duplication though. That is the same bugfix landing in each of those versions.